### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Adds support to follow other users

### DISCUSSION THREAD

**Discussion by gwealm** [dailydotdev/daily#1117] [Feature Requests]
### Concise Description

Adds support to follow other users

### Description

When going to another user's profile, it would be cool to have a button to follow that user. 
I believe this would be a good option for the user, since we already have support for creating posts, seeing other people's profiles, and other social actions. 
More details of this feature are explained below.

### Solution Proposed

 This "following" action could mean a lot of things and vary from a very simple feature to a more complex one. Still, for usability and simplicity purposes I would suggest that following another user meant that:

- I could see that users' dev card properties
- I could see the squads that the user belongs to
- I could see that users' posts
- I could see that users' upvotes

For this feature not to be intrusive to the end-user, I would like to have a variety of options on my profile page (or even on an options page) to choose what I want to show to other people.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @gui1612,

Thanks for the suggestion! 
Points taken. We will keep you posted 🙏
